android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -85,20 +88,20 @@ public WebRTCModule(ReactApplicationContext reactContext) {
                 encoderFactory = new SoftwareVideoEncoderFactory();
                 decoderFactory = new SoftwareVideoDecoderFactory();
             }
-        }
-
-        if (adm == null) {
-            adm = JavaAudioDeviceModule.builder(reactContext).setEnableVolumeLogger(false).createAudioDeviceModule();
-        }
 
```

Comment:
Forgot to close here.
Also give us a new line.

Reply:
Fixed. Also, there is a newline before the log messages:

![image](https://github.com/react-native-webrtc/react-native-webrtc/assets/77067012/7f5f543c-1496-47fe-b3c8-b498663fd836)